GiEnJul/Startup.cs

```diff
@@ -28,8 +32,27 @@ public void ConfigureServices(IServiceCollection services)
             {
                 configuration.RootPath = "ClientApp/build";
             });
+            string domain = $"https://{Configuration["Auth0:Domain"]}/";
+            string audience = $"https://{Configuration[(_env.IsDevelopment()) ? "Auth0:LocalAudience" : "Auth0:AzureAudience"]}/";
+            System.Console.WriteLine(audience);
+            services.AddAuthentication(options =>
+            {
+                options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
+                options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
+            })
+            .AddJwtBearer(options =>
+            {
+                options.Authority = domain;
+                options.Audience = audience;
+            }
+            );
```

Comment:
```suggestion
            services.AddAuthentication(options =>
            {
                options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
                options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
            })
            .AddJwtBearer(options =>
            {
                var section = Configuration.GetSection("Auth0");
                options.Authority = section.GetValue<string>("Domain");
                options.Audience = section.GetValue<string>(_env.IsDevelopment() ? "LocalAudience" : "AzureAudience");
            }
            );
```
This would require that you also add https:// to all appsettings.json variables under Auth0.

Reply:
updated with 1a4b189